packages/utils/src/crypto/packet/padding.ts

```diff
@@ -0,0 +1,33 @@
+import { PAYLOAD_SIZE } from './constants'
+import { u8aEquals } from '../../u8a'
+
+export const PADDING_TAG = Uint8Array.from([72, 79, 80, 82])
```

Comment:
where do these magic numbers come from?

Reply:
It's "HOPR" in ASCII but doesn't really matter which bitstring you pick as long as you always use the same

Reply:
added a comment